firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -314,3 +315,64 @@ uint8_t TCMD_agenda_fetch(){
 
     return 0;
 }
+
+
+/// @brief Deletes a telecommand from the agenda by the telecommand function name.
+/// @param function_name The `timestamp_sent` value of the telecommand to delete.
+/// @return 0 on success, > 0 if the telecommand was not found.
+/// @note Calls `LOG_message()` to log the deletion before all returns.
+uint8_t TCMD_agenda_delete_by_function_name(const char *function_name) {
+    bool is_valid_function_name_count = false;
+
+    // Get count of active agendas
+    const uint8_t active_agendas = TCMD_get_agenda_used_slots_count();
+
+    if(active_agendas == 0){
+        LOG_message(
+        LOG_SYSTEM_TELECOMMAND, LOG_SEVERITY_NORMAL, LOG_SINK_ALL,
+        "TCMD_agenda_delete_by_function_name: No active telecommands in the agenda."
+        );
+    return 1;
```

Comment:
Fix indentation. `return 1;` should be indented. Please be aware of proper indentation.

Reply:
Sure thing. Ill be more keen on this